src/index.html

```diff
@@ -3,11 +3,51 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="body-stars">
+      <div class="block_stars stars--0">
+        <span class="star"></span><span class="star"></span><span class="star"></span><span class="star"></span><span
+          class="star"></span>
```

Comment:
Format this code as it is done with the following blocks (they have the correct formatting)